pkg/universe.dagger.io/alpha/php/composer/image.cue

```diff
@@ -0,0 +1,44 @@
+package composer
+
+import (
+	"universe.dagger.io/docker"
+)
+
+// Composer image default name
+_#DefaultName: "composer"
+
+// Composer image default repository
+_#DefaultRepository: "index.docker.io"
+
+// Co image default version
+_#DefaultVersion: "latest"
+
+#Image: {
+	name:       *_#DefaultName | string
+	repository: *_#DefaultRepository | string
```

Comment:
See https://github.com/dagger/dagger/pull/2632#pullrequestreview-1016560235 related comment about why I think this is a bad idea to have `name` and `repository`.